react/src/components/VFolderTableFormItem.tsx

```diff
@@ -84,29 +76,43 @@ const VFolderTableFormItem: React.FC<VFolderTableFormItemProps> = ({
         {/* <Flex>{form.getFieldValue('vfoldersAliasMap')}</Flex> */}
       </Form.Item>
       <Form.Item hidden name="autoMountedFolderNames" />
-      <Form.Item
-        name={'mounts'}
-        {...formItemProps}
-        valuePropName="selectedRowKeys"
-        trigger="onChangeSelectedRowKeys"
-      >
-        <VFolderTable
-          rowKey={rowKey}
-          showAliasInput
-          aliasMap={form.getFieldValue('vfoldersAliasMap')}
-          onChangeAliasMap={(aliasMap) => {
-            form.setFieldValue('vfoldersAliasMap', aliasMap);
-            form.validateFields(['vfoldersAliasMap']);
-          }}
-          // TODO: implement pagination
-          pagination={false}
-          filter={filter}
-          showAutoMountedFoldersSection
-          onChangeAutoMountedFolders={(names) => {
-            form.setFieldValue('autoMountedFolderNames', names);
-          }}
-          {...tableProps}
-        />
+      <Form.Item noStyle dependencies={['owner']}>
+        {({ getFieldValue }) => {
+          const ownerInfo = getFieldValue('owner');
+          const ownerEmail =
+            ownerInfo?.enabled && ownerInfo?.email
+              ? ownerInfo.email
+              : tableProps?.ownerEmail;
+
+          return (
```

Comment:
The validation for ownerEmail is being validated by the parent component using the vfolderTagbleFormItem component. I don't think we need this part. The part that passes to the child props below also contains the ownerEmail inside the tableProps, so I think we can delete that as well.